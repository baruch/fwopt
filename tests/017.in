iptables -A INPUT -i eth0 -m state --state ESTABLISHED,RELATED -j ACCEPT
iptables -A INPUT -i eth1 -m state --state ESTABLISHED,RELATED -j ACCEPT
