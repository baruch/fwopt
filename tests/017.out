iptables -N chain_1
iptables -A chain_1 -i eth0 -j ACCEPT
iptables -A chain_1 -i eth1 -j ACCEPT
iptables -A INPUT --match state --state ESTABLISHED,RELATED -j chain_1
